mrscake/src/components/Reg.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { Button, Card, CardFooter, CardBody, CardGroup, Col, Container, Form, 
+    Input, InputGroup, InputGroupAddon, InputGroupText, Row } from 'reactstrap';
+class Reg extends Component {
```

Comment:
What do you mean whit Reg?

Reply:
I will change from Reg to Registration in next commit.